src/openrct2/actions/TrackPlaceAction.cpp

```diff
@@ -625,12 +625,17 @@ GameActions::Result TrackPlaceAction::Execute() const
             case TrackElemType::SpinningTunnel:
                 MapAnimationCreate(MAP_ANIMATION_TYPE_TRACK_SPINNINGTUNNEL, CoordsXYZ{ mapLoc, trackElement->GetBaseZ() });
                 break;
+            case TrackElemType::Brakes:
+                trackElement->SetBrakeClosed(true);
+                break;
         }
+
         if (TrackTypeHasSpeedSetting(_trackType))
         {
             trackElement->SetBrakeBoosterSpeed(_brakeSpeed);
         }
-        else if (ride->GetRideTypeDescriptor().HasFlag(RIDE_TYPE_FLAG_HAS_LANDSCAPE_DOORS))
+
+        if (ride->GetRideTypeDescriptor().HasFlag(RIDE_TYPE_FLAG_HAS_LANDSCAPE_DOORS))
```

Comment:
why this change? The logic is a bit silly nowadays since the seat and speed are split just want to know if this was to fix/address an issue.

Reply:
The original only allows setting seat rotation if speed is not set - this PR supports speed and seat rotation on block brakes because td6 encodes seat rotation not block brake speed.